patches/core/ungoogled-chromium/disable-translate.patch

```diff
@@ -1,38 +1,111 @@
-# Disables browser translation
-
---- a/components/translate/content/renderer/translate_agent.cc
-+++ b/components/translate/content/renderer/translate_agent.cc
-@@ -74,8 +74,7 @@ TranslateAgent::TranslateAgent(content::
-     : content::RenderFrameObserver(render_frame),
-       world_id_(world_id),
-       extension_scheme_(extension_scheme) {
--  translate_task_runner_ = this->render_frame()->GetTaskRunner(
--      blink::TaskType::kInternalTranslation);
-+  CancelPendingTranslation();
+# Disables translation and removes the "Translate to" context menu when --translate-script-url flag is not set
```

Comment:
This patch filename and this comment here is a little confusing because we don't always disable translation. Maybe we could rename the patch to something like `toggle-translation-via-switch.patch` or something to that effect?

Reply:
Makes sense to me!